oneview_redfish_toolkit/api/chassis.py

```diff
@@ -50,10 +50,11 @@ def __init__(self, oneview_resource):
         self.redfish["SerialNumber"] = oneview_resource["serialNumber"]
         self.redfish["PartNumber"] = oneview_resource["partNumber"]
         self.redfish["Status"] = collections.OrderedDict()
-        self.redfish["Status"]["State"] = \
```

Comment:
We need to update the copyrights for changed files.

Reply:
Chassis also map to server hardware, thus we can update State and Health as discussed and reviewed in the spreadsheet.